aip/general/0133.md

```diff
@@ -58,10 +58,8 @@ rpc CreateBook(CreateBookRequest) returns (Book) {
   **must** map to the resource field in the request message.
   - All remaining fields **should** map to URI query parameters.
 - There **should** be exactly one `google.api.method_signature` annotation,
```

Comment:
Why is this **should** and not **must** ?

Reply:
as a quick note: this was **should** originally, so it's actually just adjusting the terminology based on  the new required value, rather than adjusting the level of guidance.

Reply:
Ack. Can revisit should vs must separately.